generators/entity/prompts.js

```diff
@@ -983,7 +983,7 @@ function askForRelationship(done) {
                         name: 'one-to-one'
                     }
                 ];
-                if (response.otherEntityName.toLowerCase() !== 'user') {
+                if (['user', 'authority'].includes(response.otherEntityName.toLowerCase())) {
```

Comment:
Are you sure? It previously meant "if the other entity's name isn't user", and now it means "if the other entity's name is either user or authority". Haven't you forgotten a `!`?

Reply:
yeah I did >.<

Reply:
;)